src/main/java/com/igot/cb/transactional/exceptions/CassandraPropertyReaderException.java

```diff
@@ -0,0 +1,22 @@
+package com.igot.cb.transactional.exceptions;
```

Comment:
Please remove this folder transactional if there are no references

Reply:
this is required, we are using cassandra for capturing upvote and downvote data